main.css

```diff
@@ -0,0 +1,339 @@
+
+/** Breakpoints:
+ * md > 768px
+ * lg > 992px
+ */
+
+/**
+ * Main theme
+ * ======================
+ **/
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+  margin: 0;
+  padding: 0;
+}
+
+@media (min-width: 768px) {
+  body {
+    font-size: 16px;
+  }
+}
+
+.t-bg-red {
+  background-color: #f72211;
+}
+
+.t-bg-gray {
+  background-color: #e4e5e6;
+}
+
+.t-bg-verydark-gray {
+  background-color: #c2d4d7;
+}
+
+.t-bg-translucid {
+  background: rgba(255,255,255,.9);
+}
+
+.t-bg-white {
+  background-color: white;
```

Comment:
- [ ] Double check if all of your styles in this file really can't be done using Bootstrap classes instead.
Some styles you are currently using pure CSS to apply but could be easily exchanged for Bootstrap classes are:
- bg or text color white
- font-weight
- width 100%
- display
- position

Make a quick sweep on your stylesheets, check the classes you can use on the [Bootstrap documentation](https://getbootstrap.com/docs/4.1/getting-started/introduction/), and whenever possible use the Bootstrap class instead.

Reply:
By getting the full scope of what Bootstrap can do now you'll be able to style your future projects with it way faster, so doing a little digging in the documentation right now will pay really pay off.